pkg/pubsub/adapter/converters/pubsub.go

```diff
@@ -19,20 +19,27 @@ package converters
 import (
 	"context"
 
-	cloudevents "github.com/cloudevents/sdk-go"
-	cepubsub "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub"
+	"github.com/cloudevents/sdk-go"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub"
+	cepubsub "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub/context"
 	"github.com/google/knative-gcp/pkg/apis/pubsub/v1alpha1"
 )
 
-func convertPubsub(ctx context.Context, msg *cepubsub.Message, sendMode ModeType) (*cloudevents.Event, error) {
+const (
+	Publish = "google.pubsub.topic.publish"
+
+	PubSubResourceGroup = "pullsubscriptions.pubsub.cloud.run"
+)
+
+func convertPubsub(ctx context.Context, msg *pubsub.Message, sendMode ModeType) (*cloudevents.Event, error) {
 	tx := cepubsub.TransportContextFrom(ctx)
 	// Make a new event and convert the message payload.
 	event := cloudevents.NewEvent(cloudevents.VersionV03)
 	event.SetID(tx.ID)
 	event.SetTime(tx.PublishTime)
 	event.SetSource(v1alpha1.PubSubEventSource(tx.Project, tx.Topic))
 	event.SetDataContentType(*cloudevents.StringOfApplicationJSON())
-	event.SetType(v1alpha1.PubSubEventType)
```

Comment:
why not use the versioned type?

Reply:
this is the only place we are using it, so I moved it here... But if you want it in the versioned type, I can move it back...